qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -81,6 +92,61 @@ public void catchesCheckstyleViolationsInLicense() throws Exception {
     }
 
     /**
+     * CheckstyleValidator can accept instance method references.
+     * @throws Exception In case of error
+     */
+    @Test
+    public void acceptsInstanceMethodReferences() throws Exception {
+        final Environment.Mock mock = new Environment.Mock();
+        final File license = this.rule
+            .savePackageInfo(new File(mock.basedir(), DIRECTORY))
+            .withLines(new String[] {"Hello."})
+            .withEol("\n")
+            .file();
+        final String content = Joiner.on("\n").join(
+            "/**",
+            " * Hello.",
+            " */",
+            "package foo;",
+            "/**",
+            " * Simple.",
+            " * @version $Id $",
+            " * @author John Smith (john@example.com)",
+            " */",
+            "public final class Main {",
+            "    /**",
+            "     * Start. Check fails in this method.",
+            "     */",
+            "    private void start() {",
+            "        Collections.singletonList(\"1\")",
+            "            .forEach(this::doSomething);",
+            "    }",
+            "    /**",
+            "     * Method to be referenced.",
+            "     * @param value Value to print",
+            "     */",
+            "    private void doSomething(final String value) {",
+            "        System.out.println(value);",
+            "    }",
+            "}",
+            ""
+        );
+        final StringWriter writer = new StringWriter();
+        org.apache.log4j.Logger.getRootLogger().addAppender(
+            new WriterAppender(new SimpleLayout(), writer)
+        );
+        final Environment env = mock.withParam(
+            CheckstyleValidatorTest.LICENSE_PROP,
+            this.toURL(license)
+        ).withFile("src/main/java/foo/Main.java", content);
+        new CheckstyleValidator().validate(env);
+        MatcherAssert.assertThat(
+            writer.toString(),
+            Matchers.containsString("No Checkstyle violations found")
```

Comment:
@pinaf all other tests are checking that validation fails so that's why this one is different, I want to check that validation passed